test/PlaygroundTransform/playground_options.swift

```diff
@@ -0,0 +1,65 @@
+// Tests that `-playground-option` can turn on various options individually,
+// including those that are off by default for `-playground-high-performance`.
+// Also test that unknown options are gracefully ignored.
+//
+// REQUIRES: executable_test
+//
+// RUN: %empty-directory(%t)
+// RUN: cp %s %t/main.swift
+// RUN: %target-build-swift -whole-module-optimization -module-name PlaygroundSupport -emit-module-path %t/PlaygroundSupport.swiftmodule -parse-as-library -c -o %t/PlaygroundSupport.o %S/Inputs/SilentPCMacroRuntime.swift %S/Inputs/PlaygroundsRuntime.swift
+//
+// RUN: %target-build-swift -Xfrontend -playground -Xfrontend -playground-high-performance -Xfrontend -playground-option -Xfrontend FunctionParameters -Xfrontend -playground-option -Xfrontend ScopeEvents -Xfrontend -playground-option -Xfrontend ThisOptionShouldBeGracefullyIgnored -o %t/main -I=%t %t/PlaygroundSupport.o %t/main.swift
+// RUN: %target-codesign %t/main
+// RUN: %target-run %t/main | %FileCheck %s
+//
+// RUN: %target-build-swift -Xfrontend -playground -Xfrontend -playground-high-performance -Xfrontend -playground-option -Xfrontend FunctionParameters -Xfrontend -playground-option -Xfrontend ScopeEvents -Xfrontend -playground-option -Xfrontend ThisOptionShouldBeGracefullyIgnored -o %t/main2 -I=%t %t/PlaygroundSupport.o %t/main.swift
```

Comment:
Will we want to upgrade `-playground-option` to an official driver flag?

Reply:
I'm not sure — I would think so.  Would that be a separate PR on SwiftDriver?

Reply:
Having this be a driver option does sound more ergonomic.  One of the future changes I have lined up to propose as a PR is to have a way to print the available playground transform options, and that definitely feels as if it belongs in the driver (with the data coming from the frontend of course).

Reply:
We would need to move the option to `Options.td` and then yeah have a separate PR against swift-driver following these instructions:
https://github.com/apple/swift-driver#rebuilding-optionsswift

Reply:
Thank you!  I'll take a look at that.

Reply:
I've taken a closer look here, it looks to me as if all the other playground options are currently just frontend flags (including the `-playground` option itself that enables the transform).  So it looks to me as if enabling these are driver flags should be a separate PR, which would enable `-playground` as well as `-playground-option` (and ideally also a new `-playground-print-options` that would list the supported options.  @artemcm WDYT?